### ISSUE TITLE
How to connect to redis with unix domain socket?

### ISSUE THREAD

**Issue by johnrosen1** [chihaya/chihaya#595]
No docs about this

**Comment by mrd0ll4r** [comment]
Yeah, that's currently not implemented. The `redisBackend` supports this, but the redis `storage` implementation does not have configuration for a unix socket.

If you want to implement this, this is the relevant part in the code: https://github.com/chihaya/chihaya/blob/main/storage/redis/peer_store.go#L83-L193

We'd be happy to take a PR, and it shouldn't be too difficult :)

**Comment by johnrosen1** [comment]
sure
